### DISCUSSION TITLE
showing user_attrs on dashboard?

### DISCUSSION THREAD

**Discussion by zaferyumruk** [optuna/optuna-dashboard#164] [General]
Do you have any plans to display user_attrs on the dashboard? 

That feature would be of great help while examining multiple studies.

**Comment by c-bata** [comment]
Hi. You can check user_attrs of studies and trials via down-arrow buttons.

| study user_attrs | trial user_attrs |
| --- | --- |
| ![study user_attrs](https://user-images.githubusercontent.com/5564044/114265534-40b87100-9a2c-11eb-947f-02448809d8cd.gif) | ![trial user_attrs](https://user-images.githubusercontent.com/5564044/114265667-20d57d00-9a2d-11eb-8b9c-69541c9b4a28.gif) |

**Reply by zaferyumruk** [reply]
Thank you for the quick response. How could I miss that? 

optuna and optuna-dashboard are both great!
